### DISCUSSION TITLE
Digital Assets meeting agenda for 2023-02-13

### DISCUSSION THREAD

**Discussion by OBWANDO** [Open-MV/fig-digitalassets#2] [Meetings]
## Meeting Details

- **Date/Time:** February 13, 2023 @ Monday 12:00 PM ET / 17:00 UTC / 6:00 PM CET
- **Location:** Link should be posted in the `#Digital-assets` text and voice channel on Discord at least 5 days before the meeting.
- **Moderator:** TBA
- **Note Taker** Person

The [FIG Digital Assets](https://github.com/Open-MV/fig-digitalassets/tree/main/meetings) repo contains the history past meetings, including a link to the agenda, recording, notes, and resources.

## Meeting Agenda
First meeting and operating
FIG charter discussion
Database Feature triage

**Discuss agenda from proposed topics**

## Outcomes from Discussion topics

**Discuss outcomes from agenda**

## Action Items

**Create actionable items from proposed topics**

## Items for next week's meeting
**List items that need to be discussed next week**

## Open Discussion Items

List any additional items below!

**Comment by vamman** [comment]
I would like to add the following items to the discussion for the next few weeks on the format being used to transport DAP. We have a tremendous amount of experience on this topic with GenXP and our clients wants/needs. Please leverage our experience on this topic.

DAP can not exist without a universal format the captures everything that is common with a rendering scene:

- Begin discussion on use of different formats currently available such as 
- Universal Scene Descriptor (USD) as a starting point to support non-destructive digital-asset portability.
- VRML
- Others??
- Render Capture/Injection as a possible R&D topic that allows us to explore where Injection is the best method to satisfy all engines?
- Omniverse: What is it? How does it work? How can we use it?
- Where and how are DAPs stored as a binary? Public depot? How does authorship/ownership/enterprise IP of a DAP work?

Relevant links:
- UE5.1 integration with USD: https://docs.unrealengine.com/5.1/en-US/universal-scene-description-in-unreal-engine/
- Pixar USD: https://graphics.pixar.com/usd/release/index.html
- NVIDIA USD: https://developer.nvidia.com/usd
- VRML: https://en.wikipedia.org/wiki/VRML

**Comment by aaronfranke** [comment]
I would like to discuss bringing in the GLTF extension standards that OMI has been working on and having the OMF endorse them and/or iterate on them if we discuss any changes.

* Collider: https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_collider
* Physics body: https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_physics_body
* Spawn point: https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_spawn_point
* Audio: https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/KHR_audio

All of these build on top of [GLTF](https://github.com/KhronosGroup/glTF), the open 3D model format from Khronos.

I see above that there is discussion about USD. A lot of open projects are adopting GLTF, but we could support both GLTF and USD if that's desired. These standards I linked above could be ported over to USD if it supports extending with arbitrary data like GLTF does, and then that would provide an easy path to converting between the two.

**Comment by LaylBongers** [comment]
I would like to briefly ask people to look at the Dacti package format draft at this point and give feedback.
<https://gist.github.com/LaylBongers/4981bdbc4daa05b790991d37a429c733>
